test/unit/behaviours/tenancy-start-west-midlands-check.test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+const reqres = require('reqres');
+const Behaviour = require('../../../apps/right-to-rent-check/behaviours/tenancy-start-west-midlands-check');
+
+describe('apps/behaviours/tenancy-start-west-midlands-check', () => {
+  it('exports a function', () => {
+    expect(Behaviour).to.be.a('function');
+  });
+
+  describe('getNextStep()', () => {
+    class Base {
+      getNextStep() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let ValidTenants;
+
+    beforeEach(() => {
+      sessionModel = {
+        get: sinon.stub()
+      };
+      res = reqres.res();
+      req = reqres.req({sessionModel});
+      sinon.stub(Base.prototype, 'getNextStep');
+      ValidTenants = Behaviour(Base);
+      instance = new ValidTenants();
+    });
+
+    afterEach(() => {
+      Base.prototype.getNextStep.restore();
+    });
+
+    it('calls super getNextStep if valid-tenancy is true', () => {
+      req.sessionModel.get.withArgs('valid-tenancy').returns(true);
+      instance.getNextStep(req, res);
+
+      Base.prototype.getNextStep.should.have.been.calledWith(req, res);
+    });
+    it('return /exit page if valid-tenancy is false', () => {
+      req.sessionModel.get.withArgs('valid-tenancy').returns(false);
+      const result = instance.getNextStep(req, res);
+      result.should.equal('/exit-dont-need-check');
+    });
+  });
+  describe('saveValues()', () => {
+    class Base {
+      saveValues() {}
+    }
+    let req;
+    let res;
+    let sessionModel;
+    let instance;
+    let ValidTenants;
+    let callback;
+
+    beforeEach(() => {
+      sessionModel = {
+        get: sinon.stub(),
+        set: sinon.stub()
+      };
+      res = reqres.res();
+      req = reqres.req({sessionModel});
+      callback = sinon.stub();
+
+      sinon.stub(Base.prototype, 'saveValues');
+      ValidTenants = Behaviour(Base);
+      instance = new ValidTenants();
+    });
+
+    afterEach(() => {
+      Base.prototype.saveValues.restore();
+    });
+
+    it('if the tenancyStart is after 2016-01-31 then do NOT set valid-tenancy to false', () => {
```

Comment:
By default valid-tenancy is set to true in the behaviour.  This makes the code smaller and neater.  Otherwise, the code gets a lot of if then and elses